pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -22,6 +22,21 @@ const (
 	auditPolicyAsset = "pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml"
 )
 
+// DefaultPolicy brings back the default.yaml audit policy to init the api
+func DefaultPolicy() ([]byte, error) {
+	// none is used on target name and namespace as it's not a key in the default.yaml
+	cm, err := GetAuditPolicies("none", "none")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retreive audit policies - %w", err)
+	}
+
+	cmData := []byte(cm.Data["default.yaml"])
+	if len(cmData) == 0 {
+		return nil, errors.New("failed to locate the default policy from configmap")
```

Comment:
can this happen? Can't we make sure with a test that `default.yaml` always available?

Reply:
Test is there previously:
https://github.com/openshift/library-go/blob/master/pkg/operator/apiserver/audit/audit_policies_test.go#L102

Reply:
I think I can live with this. I would have preferred no error in the signature, but that's just taste.